torchvision/transforms/v2/_misc.py

```diff
@@ -124,7 +174,26 @@ def _transform(
         return F.normalize(inpt, mean=self.mean, std=self.std, inplace=self.inplace)
 
 
+# TODO: This should be in the _color.py file!
```

Comment:
I don't feel it belongs there, but no strong opinion.

Reply:
But the transform is in under the "color" section in the docs?

Reply:
Oh, in that case I missed to flag this there.